tests/acceptance/common.py

```diff
@@ -248,9 +248,53 @@ def qemu_finalizer_impl():
 
 @pytest.fixture(scope="function")
 def no_image_file(qemu_running):
+    """Make sure 'image.dat' is not present on the device."""
     execute(no_image_file_impl, hosts=conftest.current_hosts())
 
 
 def no_image_file_impl():
     run("rm -f image.dat")
 
+def latest_build_artifact(extension):
+    if len(os.environ['BUILDDIR']) <= 0:
```

Comment:
a more pythonic way of doing this would be:

`os.environ.get("BUILDDIR", False)`